src/plugins/discover/public/context_awareness/profiles_manager.ts

```diff
@@ -130,6 +134,7 @@ export class ProfilesManager {
       return;
     }
 
+    this.trackActiveProfiles(this.rootContext$.getValue().profileId, context.profileId);
```

Comment:
@davismcphee I don't think that it would be helpful to report a default `default-data-source-profile` in this case. Better to track the active profiles when users actually access their data and we know for sure what data source was it. Then we would have a cleaner reporting regarding the usage of one profile versus another.

Reply:
Fair enough 👍 It's easy enough change later on anyway if we ever identify a need for this